src/fileVersionManagement.js

```diff
@@ -67,4 +57,61 @@ FilesVersions.prototype.listVersions = function (fileId) {
   })
 }
 
+FilesVersions.prototype.getFileVersionContents = function (fileId, versionId) {
+  const path = '/meta/' + fileId + '/v/' + versionId
+
+  return new Promise((resolve, reject) => {
+    // TODO: use davclient ?
+    helpers._get(helpers._buildFullWebDAVPathV2(path)).then(data => {
+      var response = data.response
+      var body = data.body
+
+      if (response.statusCode === 200) {
+        resolve(body)
+      } else {
+        var err = helpers._parseDAVerror(body)
+        reject(err)
+      }
+    }).catch(error => {
+      reject(error)
+    })
+  })
+}
+
+FilesVersions.prototype.restoreFileVersion = (fileId, versionId, targetPath) => {
+  return new Promise((resolve, reject) => {
+    if (!helpers.getAuthorization()) {
```

Comment:
oh, we have a stateful global helper class ?

can we rather encapsulate this into a class and inject it in the constructor ? (separate PR)

Reply:
The helpers Class is injected in the ctor